src/test/java/com/epam/tc/hw3/tests/TestHomePageControlElements.java

```diff
@@ -0,0 +1,108 @@
+package com.epam.tc.hw3.tests;
```

Comment:
Also I don't see your implementation for Page Components

Reply:
Done